config/routes.rb

```diff
@@ -2,4 +2,6 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :trips, only: [ :create, :show, :index ]
```

Comment:
I don't see how :index is necessary for the purpose of the story of this pull request.

Reply:
no it`s not. we forgot to delete. it. Was just for testign purposes